apps/core/app/[locale]/(default)/login/register-customer/page-data.ts

```diff
@@ -0,0 +1,105 @@
+import { cache } from 'react';
+
+import { getSessionCustomerId } from '~/auth';
+import { client } from '~/client';
+import { FORM_FIELDS_FRAGMENT } from '~/client/fragments/form-fields';
+import { graphql, VariablesOf } from '~/client/graphql';
+import { revalidate } from '~/client/revalidate-target';
+
+const RegisterCustomerQuery = graphql(
```

Comment:
I believe we can use fragment collocation better, similar to https://github.com/bigcommerce/catalyst/pull/848, but not sure if it should be part of this effort.